cmd/minikube/cmd/start.go

```diff
@@ -172,6 +173,15 @@ func runStart(cmd *cobra.Command, args []string) {
 	console.OutStyle(console.Happy, "minikube %s on %s (%s)", version.GetVersion(), runtime.GOOS, runtime.GOARCH)
 	validateConfig()
 
+	currentUser, err := user.Current()
+
+	// Display warning if minikube is being started with root and vmDriver is not HyperV
+	if err != nil {
+		glog.Errorln("Error getting the current user: ", err)
```

Comment:
please always use %v for errror.
like glog.Errorf(""Error getting the current user %v", err)